### PR TITLE
[T3A4][W13-A1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**
Handling exceptions

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Ready to review

File: src/seedu/addressbook/storage/StorageFile.java [hunk 1]
```diff
@@ -138,7 +138,9 @@ public AddressBook load() throws StorageOperationException {
             throw new StorageOperationException("Error parsing file data format");
         } catch (IllegalValueException ive) {
             throw new StorageOperationException("File contains illegal data values; data type constraints not met");
-        }
+        }catch (Exception e){
+           e.printStackTrace();
```
Comment (okkhoy) [inline src/seedu/addressbook/storage/StorageFile.java]: printing stack trace is not an acceptable "exception handling" mechanism :-P

**Comment by okkhoy** [comment]
you are throwing an exception in the `load` method; if the `addressbook.txt` file is not present, the way I see it, your program may not start (of course, I haven't looked at the entire code, so may be I am not 100% right) 

recollect the discussion we had during tutorial and think which method would be the best place to 
1. throw the exception
2. catch and handle it.

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Some comments added. Please ack & close the PR after reading comments.

**Comment by LuMiN0uSaRc** [comment]
Noted with thanks!

**Pull request closed without merging**